openmdao/docs/feature_reference/working_with_models/debugging/om_command.rst

```diff
@@ -0,0 +1,120 @@
+.. _om-command:
+
+******************
+Command Line Tools
+******************
+
+OpenMDAO has a number of debugging/viewing command line tools that are available via the `om`
+command.  There are two types of commands available, those that perform some sort of viewing or
+configuration checking on the Problem after its setup is complete, and those that are used to
+collect information about the entire run of the Problem, things like profilers and tracers.
+
+
+Post-setup Commands
+-------------------
+
+The following commands all run immediately after a Problem's final_setup function.  After they
+complete, the program will exit, rather than continuing to the end of the user's run script.
```

Comment:
This seems unclear in that the commands themselves don't run after `setup`...  the commands  are actually the start of the process.. they run the script up to the final_setup and then hijack/intercept execution to run the desired function.  (The function runs after final setup, but the "command" embodies the entire process.)